renderer/pages/index.tsx

```diff
@@ -594,7 +594,7 @@ const Home = () => {
                 zoomAmount={zoomAmount}
                 setZoomAmount={setZoomAmount}
                 resetImagePaths={resetImagePaths}
-                hideZoomOptions={true}
```

Comment:
> When using a boolean attribute in JSX, you can set the attribute value to true or omit the value. This helps to keep consistency in code.

Sure, but please remember to also mention this in the PR description or propose them in a separate PR next time.